cli/src/command/account.rs

```diff
@@ -919,40 +919,68 @@ async fn print_address(account: &Account, address: &AccountAddress) -> Result<()
     let addresses = account.addresses_with_unspent_outputs().await?;
     let current_time = iota_sdk::utils::unix_timestamp_now().as_secs() as u32;
 
+    let mut output_ids = &Vec::new();
```

Comment:
Bit weird. You're putting new lines in the log do you could just print in the if and not do this

Reply:
not exactly sure what you mean tbh. Can you elaborate?

Reply:
It looks like you're doing this weird thing for the log at the end but you could log it earlier and have the exact same result

Reply:
still not sure what the weird thing is you're talking about :sweat_smile: , I basically(!) kept the log formatting as it was .. I just added the assets to it, no?

Reply:
You're taking a ref of a new Vec but it's not necessary at all

Reply:
see .. the way it was before ... you would have to edit 2 formatted strings (and keep them in sync). I consolidated that into just 1. That's why it has moved out of the if block.

Reply:
Idk, looks like it could be reworked and logged individually since we new line anyway

Reply:
I can change it so they are individually logged if you want. As I said ... this is how it was done before and I just kept it :shrug:

Reply:
Lol...Gh didnt show me the text you wrote about the &Vec::new 🙈 when I replied. Yeah that's weird, but a relatively small price to pay and compiler checked contrary to maintaining 2 formatted strings.

Reply:
not sure you're okay with this now ... feels like there was something else you didn't like with the logging (but as I said nothing I fundamentally changed with this PR). But I prefer this solution over having another format string with hard-coded `[]`s and `0` in case the if-block isn't executed. The downside of course are the allocations but no reason to optimize this part of the code IMO.